contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -5276,56 +5276,147 @@ transform_pivot_clause(ParseState *pstate, SelectStmt *stmt)
 }
 
 
-static Node* optimize_explicit_cast(ParseState *pstate, Node *node)
+static inline bool
+is_integer_type(Oid type) {
```

Comment:
style - starting bracket `{` in a new line, e.g.,
```
is_integer_type(...)
{
    ....
}
```

Reply:
Fixed.